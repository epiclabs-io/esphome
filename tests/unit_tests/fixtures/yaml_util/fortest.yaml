---
substitutions:
  pins: [14, 15, 33, 39]

esphome:
  name: test

binary_sensor: !for
  items: ${pins}
  var: pin
  repeat:
    platform: gpio
    pin: ${pin}
