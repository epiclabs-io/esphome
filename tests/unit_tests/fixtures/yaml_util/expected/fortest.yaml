binary_sensor:
- pin: 14
  platform: gpio
- pin: 15
  platform: gpio
- pin: 33
  platform: gpio
- pin: 39
  platform: gpio
esphome:
  name: test
